components/dropdown/test/dropdown-content.test.js

```diff
@@ -100,8 +100,8 @@ describe('d2l-dropdown', () => {
 		it('sets scrollTop to specified value', async() => {
 
 			content.boundary = {
-				below: 1,
```

Comment:
This change relates to how scaling is handled. Because the header and footer are accounted for in the calculations now, at very small sizes scrolling won't initiate for the content space (think 1-10 pixels total height for the content area and there isn't room for that) but this is a non-issue because the full dropdown will still appear, it may just be off-screen and require scrolling the window to see it.